src/main/java/ch/njol/skript/effects/EffKnockback.java

```diff
@@ -0,0 +1,93 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.util.Direction;
+import ch.njol.util.Kleenean;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.event.Event;
+import org.bukkit.util.Vector;
+import org.eclipse.jdt.annotation.Nullable;
+
+
+@Name("Knockback")
+@Description("Knocks a living entity in a direction. Mechanics such as knockback resistance will be factored in.")
+@Examples({"knockback player north",
+	"knock victim (vector from attacker to victim) with strength 10"})
+@Since("INSERT VERSION")
+@RequiredPlugins("Paper 1.19.2+")
+public class EffKnockback extends Effect {
+
+	static {
+		if (Skript.methodExists(LivingEntity.class, "knockback", double.class, double.class, double.class))
+			Skript.registerEffect(EffKnockback.class, "knock[back] %livingentities% %direction% [with (strength|force) %-number%]");
```

Comment:
```suggestion
			Skript.registerEffect(EffKnockback.class, "(apply knockback to|knock[back]) %livingentities% %direction% [with (strength|force) %-number%]");
```
I just feel like this is appropriate, I just want the syntax to have apply as it's applying vectors to entities.

Reply:
Also I believe you can make %direction% a default expression with `[%direction%]`. I know other expressions have done this before, it should apply the direction in a damage event, if not for damage an event value for that could be added to allow for simple undefined direction here.

Direction for user usability is poor in my opinion, it's on a todo list to improve.

Reply:
oh good syntax suggestion
I thought it was a little clunky but didn't have anything better